README.md

```diff
@@ -27,6 +27,9 @@ To learn about limitations and differences between Cloud Storage FUSE and POSIX
 
 For information about pricing for Cloud Storage FUSE, see https://cloud.google.com/storage/docs/gcs-fuse#charges.
 
+## Agreement on Flag Compatibility
```

Comment:
I dont think we should put this in the main readme page. Its not that important imo to be put it in first page. This will be visible in the repo page as soon as someone wants to use fuse. 
Can we add it in the usage or any other readme file.